infra/vars.tf

```diff
@@ -3,6 +3,10 @@ variable "aws_region" {
   default = "eu-west-1"
 }
 
+variable "aws_account_id" {
+  type    = string
+}
```

Comment:
We do account id as a variable or we use aws_caller_identity? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity